volumes/vds/readme.md

```diff
@@ -0,0 +1,255 @@
+# Vehicle Detector System (VDS) data  
+
+VDS data is pulled daily from ITS Central database on terminal server by dags running on Morbius.  
+VDS system consists of:  
+    Division 2:  
+        -RESCU loop detectors  
+        -Blue City VDS  
+    Division 8001: (Only 1 day sample pulled)  
+        -Intersection signal detectors + Special Detectors (Special Function and Preemption detectors)  
+
+# Table Structure  
+## vds.raw_vdsdata
+This table contains parsed data from ITSC public.vdsdata. 
+Volumes are in vehicles per hour for the 20 sec bin. To convert to 15 minute volume, group by datetime_15min and take `SUM(volume_veh_per_hr) / 4 / 45` where / 4 represents hourly to 15 min conversion and / 45 represents number of 20 sec bins in a 15 minute period. This assumes both missing bins and zero values are zeros, in line with old pipeline. 
```

Comment:
I think the word volume is more common for hourly volumes and it might be better to refer to the 15-min data as 15-minute counts not volumes to avoid confusion.